### PR TITLE
Created helper code file

### PR DESCRIPTION
**Pull request by jeroenvanbaar**
helpers.py now contains functions from downloader.py and segment_transcripts.py
Paxton: some of these functions had default arguments that were unknown variables; I removed these defaults but feel free to add values back in

### CODE REVIEW

**Review by jeremymanning — CHANGES_REQUESTED** [review]
can you please restore code/downloader.py and also move the notebooks back to code instead of code/notebooks?  then i think it's ready to merge1

**Comment by jeremymanning** [comment]
note: you can just push your changes to your local fork and the pull request will update automatically (no need to close the request)

**Comment by jeroenvanbaar** [comment]
The helpers file now contains all downloader code, so no need for that file anymore right?

**Comment by jeremymanning** [comment]
This can't be merged automatically... Looks like there's a merge conflict.  @jeroenvanbaar can you resolve it on your own?  Happy to help if needed.

**Comment by jeroenvanbaar** [comment]
Didn’t have time today but will fix this tomorrow, might need your help @jeremymanning

File: code/helpers.py [hunk 1]
```diff
@@ -55,14 +73,21 @@ def wipe_formatting(script, rehtml=False):
...
+
 def cleanup_text(transcript):
     lower_nopunc = re.sub("[^\w\s.]+", '', transcript.lower())    # remove all punctuation except periods (deliminers)
     no_digit = re.sub(r"(\d+)", lambda x: num2words(int(x.group(0))), lower_nopunc)    # convert digits to words
     spaced = ' '.join(no_digit.replace(',', ' ').split())    # deal with inconsistent whitespace
     return spaced
 
 
-def get_windows(transcript, wsize=video_wsize):
+def get_windows(transcript, wsize):
```
Comment (paxtonfitzpatrick) [inline code/helpers.py]: can you add `wsize=50` as default?

File: code/helpers.py [hunk 2]
```diff
@@ -73,7 +98,7 @@ def get_windows(transcript, wsize=video_wsize):
     return video_w
 
 
-def topic_model(transcript, vec_params=vectorizer_params, sem_params=semantic_params, return_windows=False):
+def topic_model(transcript, vec_params, sem_params, return_windows=False):
```
Comment (paxtonfitzpatrick) [inline code/helpers.py]: can you add `vec_params=None` and `sem_params=None` as default, and then add this code just before the `traj = fit_transform(...` line?
```
if not vec_params:
    vec_params = {
        'model' : 'CountVectorizer', 
        'params' : {
            'stop_words' : sw.words('english')
        }
    }

if not sem_params:
    sem_params = {
        'model' : 'LatentDirichletAllocation', 
        'params' : {
            'n_components' : 100,
            'learning_method' : 'batch',
            'random_state' : 0,
        }
    }
```

File: code/helpers.py [hunk 3]
```diff
@@ -55,14 +73,21 @@ def wipe_formatting(script, rehtml=False):
     return soup
 
 
+def wipe_acting_instructions_from_dialogue(line):
```
Comment (paxtonfitzpatrick) [inline code/helpers.py:76]: can you describe (maybe add comments to code) what this function does?  I ran it on a sample script and the only difference I could see was that it removes text in parentheses, though I didn't look super closely.

**Review by paxtonfitzpatrick — CHANGES_REQUESTED** [review]
hey @jeroenvanbaar, thanks for this! I requested two changes that'll make the functions work correctly, and also had one question.  After those changes and @jeremymanning's requests, this will be a super helpful file to have for running code on the cluster, so thank you!!

**Pull request merged**